sonarqube/assets/configuration/spec.yaml

```diff
@@ -89,6 +89,54 @@ files:
             type: array
             items:
               type: string
+      - name: components_discovery
+        description: |
+          The autodiscovery patterns for components.
+          The following example collects all defect metrics for those components with names starting with "my_project"
+          and only the metrics from the "issues" category for projects starting with "issue".
+          It also excludes projects starting with "temp" from autodiscovery.
+          The maximum total number of projects processed by autodiscovery is 10.
+          
+            components_discovery:
+              limit: 10
+              include:
+                my_project*:
+                issue*:
+                  include:
+                    - issues.
+              exclude:
+                - temp*
+        value:
+          type: object
+          properties:
+          - name: limit
+            description: |
+              Maximum number of components to be 'autodiscovered'
```

Comment:
This does nothing. In next PR (before freeze) we should remove these and do either of:

- add to main description
- define like https://github.com/DataDog/integrations-core/blob/27f2de1bab1547b606ce25ae6c9594b52cdc0aef/postgres/assets/configuration/spec.yaml#L303-L327